src/middlewares/putValidation.js

```diff
@@ -0,0 +1,10 @@
+import { INVALID_BODY } from '../constants/messages';
+
+export const putValidation = (req, res, next) => {
+    const { name, phoneNum, address } = req.body;
+    if (!name || !phoneNum || !address || !Object.keys(address).includes('country' && 'street' && 'state' && 'city' && 'location' && 'postalCode')) {
```

Comment:
this is good, we can probably slowly move towards using a validation library for any future middlewares

Reply:
^__^. I will try to use lodash.get.
This is the library what you are saying right?

Reply:
> ^__^. I will try to use lodash.get.
> This is the library what you are saying right?

recommend something like this - https://joi.dev/api/?v=17.4.0